style.css

```diff
@@ -11,7 +11,7 @@ body {
   align-items: center;
   justify-content: center;
   background: #e5e5e5;
-  font-family: "Poppins", sans-serif;
+  font-family: Poppins, sans-serif;
```

Comment:
Hey @May126-sone, you're doing a fantastic job fixing the required changes 🥳. Kindly fix the images that aren't displaying and add a background to match the Figma design.

### Your design:
![image](https://github.com/May126-sone/FIGMA-1-PORTFOLIO/assets/113283174/a0587fd3-7a1a-4d03-a8e3-392775b200c1)

### Figma design:
![image](https://github.com/May126-sone/FIGMA-1-PORTFOLIO/assets/113283174/ab7556dc-66ca-49b7-9ad6-2972c5c9d1d7)